openai/api_resources/audio.py

```diff
@@ -101,14 +101,14 @@ def transcribe_raw(
     @classmethod
     def translate_raw(
         cls,
-        model,
-        file,
-        filename,
-        api_key=None,
-        api_base=None,
-        api_type=None,
-        api_version=None,
-        organization=None,
+        model: str,
+        file: IO,
+        filename: str,
```

Comment:
Here `file` has type `IO` and therefore has the filename. I think that considering this, the `filename` parameter should be deprecated.